### PR TITLE
Add hosted zone with proper domains

### PR DESCRIPTION
**Pull request by botzill**

### CODE REVIEW

File: eu-west-1/dns-parameters.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "ParameterKey": "HostedZoneName",
```
Comment (styk-tv) [inline eu-west-1/dns-parameters.json]: This is not required.

File: eu-west-1/dns.json [hunk 2]
```diff
@@ -0,0 +1,266 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Madcore-Cluster DNS setup",
+  "Parameters": {
+    "HostedZoneName": {
```
Comment (styk-tv) [inline eu-west-1/dns.json]: This is not required

File: eu-west-1/dns-parameters.json (hunk 1 above)
Comment (botzill) [reply eu-west-1/dns-parameters.json]: Fixed.

File: eu-west-1/dns.json (hunk 2 above)
Comment (botzill) [reply eu-west-1/dns.json]: Fixed.

**Pull request merged**